src/base-component/BaseComponent.jsx

```diff
@@ -1,9 +1,6 @@
 import React from 'react';
 
-import CookiePolicyBanner from '@edx/frontend-component-cookie-policy-banner';
-import { getConfig } from '@edx/frontend-platform';
```

Comment:
Is this method solely intended for the cookie policy banner or is it used for other purposes as well?